Dockerfile

```diff
@@ -1,44 +1,75 @@
-FROM python:3.13-alpine3.21
+FROM python:3.13-alpine3.21 AS base
+WORKDIR /opt/recipes
 
-#Install all dependencies.
-RUN apk add --no-cache postgresql-libs postgresql-client gettext zlib libjpeg libwebp libxml2-dev libxslt-dev openldap git
+# Print all logs without buffering: https://stackoverflow.com/a/59812588
+ENV PYTHONUNBUFFERED=1
 
-#Print all logs without buffering it.
-ENV PYTHONUNBUFFERED 1
+# Install all dependencies.
+RUN apk add --no-cache \
+  gettext \
+  git \
+  libjpeg \
+  libwebp \
+  libxml2-dev \
+  libxslt-dev \
+  openldap \
+  postgresql-client \
+  postgresql-libs \
+  zlib
 
-ENV DOCKER true
+FROM base AS deps
 
-#This port will be used by gunicorn.
-EXPOSE 8080
+ENV DOCKER=true
```

Comment:
Where is this env used, and for what purpose? didn't find it searching the source